### PR TITLE
Add org.nccmp.nccmp

### PR DESCRIPTION
**Pull request by rsmz**

### CODE REVIEW

File: org.nccmp.nccmp.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        {
+          "type": "git",
+          "url": "https://gitlab.com/remikz/nccmp",
+          "branch": "1.8.6.2"
+        }
+      ],
+      "modules": 
+      [
+        {
+			    "name": "hdf5",
```
Comment (TingPing) [inline org.nccmp.nccmp.json]: Minor whitespace issues throughout.

**Comment by TingPing** [comment]
Seems fine overall. It doesn't have an appdata file which is fine since its just a CLI-only tool.

**Comment by nedrichards** [comment]
bot, build org.nccmp.nccmp

**Comment by nedrichards** [comment]
It absolutely still needs an appdata file. We will use them in future to make CLI tools more discoverable on the website.

**Comment by LeandroStanger** [comment]
bot, build org.nccmp.nccmp

**Comment by LeandroStanger** [comment]
bot, build org.nccmp.nccmp

**Comment by LeandroStanger** [comment]
bot, build org.nccmp.nccmp

**Pull request closed without merging**